lib_xud/doc/rst/resource_usage.rst

```diff
@@ -7,73 +7,78 @@ sections.
     +-------+-------+------+-------+-------+--------+
     | X0D04 |       | P4B0 | P8A2  | P16A2 | P32A22 |
     +-------+-------+------+-------+-------+--------+
-    | X0D05 |       | P4B1 | P8A3  | P16A3 | P32A23 |
+    | X0D13 |       | P4B1 | P8A3  | P16A3 | P32A23 |
     +-------+-------+------+-------+-------+--------+
-    | X0D06 |       | P4B2 | P8A4  | P16A4 | P32A24 |
+    | X0D22 |       | P4B2 | P8A4  | P16A4 | P32A24 |
     +-------+-------+------+-------+-------+--------+
-    | X0D07 |       | P4B3 | P8A5  | P16A5 | P32A25 |
+    | X0D23 |       | P4B3 | P8A5  | P16A5 | P32A25 |
     +-------+-------+------+-------+-------+--------+
-    | X0D08 |       | P4A2 | P8A6  | P16A6 | P32A26 |
+    | X0D34 |       | P4A2 | P8A6  | P16A6 | P32A26 |
     +-------+-------+------+-------+-------+--------+
     | X0D09 |       | P4A3 | P8A7  | P16A7 | P32A27 |
     +-------+-------+------+-------+-------+--------+
-    | X0D23 | P1H0  |                               |
-    +-------+-------+------+-------+-------+--------+
-    | X0D25 | P1J0  |                               |
+    | X0D10 | P1C0  |                               |
     +-------+-------+------+-------+-------+--------+
-    | X0D26 |       | P4E0 | P8C0  | P16B0 | P32A28 |
+    | X0D12 | P1E0  |                               |
     +-------+-------+------+-------+-------+--------+
-    | X0D27 |       | P4E1 | P8C1  | P16B1 | P32A29 |
+    | X0D13 | P1F0  |                               |
     +-------+-------+------+-------+-------+--------+
-    | X0D28 |       | P4F0 | P8C2  | P16B2 |        |
+    | X0D14 |       | P4C0 | P8B0  | P16A8 |        |
     +-------+-------+------+-------+-------+--------+
-    | X0D29 |       | P4F1 | P8C3  | P16B3 |        |
+    | X0D15 |       | P4C1 | P8B1  | P16A9 |        |
     +-------+-------+------+-------+-------+--------+
-    | X0D30 |       | P4F2 | P8C4  | P16B4 |        |
+    | X0D16 |       | P4D0 | P8B2  | P16A10|        |
     +-------+-------+------+-------+-------+--------+
-    | X0D31 |       | P4F3 | P8C5  | P16B5 |        |
+    | X0D17 |       | P4D1 | P8B3  | P16A11|        |
     +-------+-------+------+-------+-------+--------+
-    | X0D32 |       | P4E2 | P8C6  | P16B6 | P32A30 |
+    | X0D18 |       | P4D2 | P8B4  | P16A12|        |
     +-------+-------+------+-------+-------+--------+
-    | X0D33 |       | P4E3 | P8C7  | P16B7 | P32A31 |
+    | X0D19 |       | P4D3 | P8B5  | P16A13|        |
     +-------+-------+------+-------+-------+--------+
-    | X0D34 | P1K0  |                               |
+    | X0D20 |       | P4C2 | P8B6  | P16A14|        |
     +-------+-------+------+-------+-------+--------+
-    | X0D36 | P1M0  |      | P8D0  | P16B8 |        |
+    | X0D21 |       | P4C3 | P8B7  | P16A15|        |
     +-------+-------+------+-------+-------+--------+
-    | X0D37 | P1N0  |      | P8C1  | P16B1 |        |
+    | X0D23 | P1H0  |                               |
```

Comment:
`X0D23` is in this table twice with different ports.

Reply:
Very good spot!
(Turns out there are many issues with this table)